frontend/src/components/workspaces/Templates/TemplatesTableLogic/TemplatesTableLogic.tsx

```diff
@@ -119,13 +114,14 @@ const TemplatesTableLogic: FC<ITemplateTableLogicProps> = ({ ...props }) => {
   } = useWorkspaceTemplatesQuery({
     variables: { workspaceNamespace },
     onCompleted: setDataTemplate,
+    fetchPolicy: 'network-only',
   });
 
   const { instances } = dataInstances?.instanceList ?? {};
 
   useEffect(() => {
     if (!loadingTemplate) {
-      subscribeToMoreTemplates({
+      const unsubscribeToMoreTemplates = subscribeToMoreTemplates({
```

Comment:
Is *unsubscribe* correct?

Reply:
Yes, but then i didn't use it. I'm going to remove it